index2.html

```diff
@@ -0,0 +1,360 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style2.css" />
+
+    <title>David Underwood Home Page</title>
+  </head>
+  <body>
+    <header class="header1">
+      <div class="header">
+        <img class="cell-info" src="Light.png" alt="" />
+      </div>
+
+      <div class="bottom-part">
+        <div class="welcome">
+          <a href="https://github.com/david63011">Welcome</a>
+        </div>
+        <img class="union" src="Union.png" alt="Hamburger" />
+      </div>
+    </header>
+
+    <section class="main-part">
+      <img class="background-photo1" src="background.png" alt="" />
+      <img class="background-photo2" src="desktop.svg" alt="" />
+      <h2 class="porto">Portoflio</h2>
+      <h2 class="about-top">About</h2>
+      <h2 class="con">Contact</h2>
+
+      <div class="greeting">
+        <h3 class="hi">Hey there. I'm</h3>
+        <h3 class="first-name">David</h3>
+        <h3 class="job">I am a software</h3>
+        <h3 class="job-info">Developer</h3>
```

Comment:
- [ ] Great job fixing some of your website accessibility issues 👏 . Kindly fix the color contrast issue in this text 🙏 

![image](https://user-images.githubusercontent.com/74814780/188688418-51e1b692-470c-41d2-b4c0-d09af0d333df.png)

Reply:
@Ugboaja-Uchechi ok so would be #36A57E I am using a tool that says large text passes the minimum requirements

Reply:
Hi @david63011, try installing [Wave evaluation tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en) as an extension on chrome. Once it's installed click on it this would help you better to know if your site has any accessibility issues. Good luck 😊